src/Sulu/Bundle/MediaBundle/Controller/MediaPreviewController.php

```diff
@@ -94,6 +110,18 @@ public function postAction($id, Request $request)
             $mediaEntity->setPreviewImage($previewImage->getEntity());
             $this->mediaManager->addFormatsAndUrl($media);
 
+            // Because the `MediaManager::save()` method calls `$entityManager->flush()` itself, the `created` event of
+            // the preview image and the `preview_image_created`/`preview_image_modified` event are not in the same badge.
```

Comment:
```suggestion
            // the preview image and the `preview_image_created`/`preview_image_modified` event are not in the same batch.
```

dont think this is critical, sguess we will just live with it 🙂